bridge-aurora-token-factory/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "bridge-aurora-token-factory"
+version = "0.1.5"
+authors = ["Near Inc <hello@near.org>"]
+edition = "2018"
```

Comment:
why not 2021?

Reply:
Fixed in f005b19c3e73964e102018e69c462aaeffa9d7eb